CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.8.1
+
+* Migration for more than one interpolation or expressions in a calc function parameter.
```

Comment:
Can you add a subheading here to note this is for the calc interpolation migrator? (same as the 1.8.0 changelog entry)

Reply:
Done!